extension/content_script.js

```diff
@@ -298,12 +298,15 @@ function highlightKeyWords(container, start, end) {
 	let wordRegex = /\b\w+\b/g;
 	let wordList = sentenceText.match(wordRegex);
 	let keywords = doc.getKeyWords();
+	// Look where keyword is in sentence AFTER last search. Guaranteed to be after. Init to 0. 
+	let keyword_search_start_pointer = 0; 
 	for (var i in wordList) { // Accentuate keywords
 		// TODO: This only gets the first occurence of each word in the sentence; should get all
 		let word = wordList[i];
 		if (keywords.has(word.toLowerCase())) { // See if each word is a keyword
-			var word_start = containerText.indexOf(word, start);
+			var word_start = containerText.indexOf(word, start+keyword_search_start_pointer);
 			var word_len = word.length;
+			keyword_search_start_pointer += word_len;
```

Comment:
I think this should be word_start + word_len.

Concerns w/ current approach:
1. Undercount the actual search_start_pointer, cos it doesn't account for the word lens of the non-matching words.
E.g. 
"match abasdfasdfasdf match"
I think by the second "match" ur keyword_ssearch_start_poniter is pointing to somewhere in the middle of the random string

2. undercounting cos ur not counting the word boundary chars.

Reply:
Good reasoning